src/genie/libs/parser/iosxe/show_ap.py

```diff
@@ -295,3 +295,680 @@ def cli(self, output=None):
+        # ab22-cap16                    5c50.1504.89a0  1       Enabled        Up            20     *7/8 (8 dBm)    (48)*
+        # ab22-cap24                    5c50.1504.89e0  1       Enabled        Up            20     *7/8 (8 dBm)    (36)*
+        # ab22-cap18                    5c50.1504.8ac0  1       Enabled        Up            20     *7/8 (6 dBm)    (124)*
+        # ab32-cap13                    5c50.1504.8bc0  1       Enabled        Up            20     *6/8 (9 dBm)    (124)*
+        #
+        # AP Name                           Mac Address     Slot    Admin State    Oper State    Width  Txpwr           Channel
+        # ---------------------------------------------------------------------------------------------------------------------------------
+        # ab22-cap10                    5c50.1504.8be0  1       Enabled        Up            20     *6/8 (9 dBm)    (132)*
+        # ab11-cap18                    5c50.1504.8c20  1       Enabled        Up            20     *7/8 (8 dBm)    (48)*
+        # ab22-cap8                     5c50.1504.8c40  1       Enabled        Up            20     *7/8 (8 dBm)    (149)*
+        # ab22-cap22                    5c50.1504.8ce0  1       Enabled        Up            20     *8/8 (5 dBm)    (40)*
+        # ab22-cap6                     5c50.1504.8d80  1       Enabled        Up            20     *7/8 (6 dBm)    (100)*
+        # ab31-cap29                    5c50.1504.8dc0  1       Enabled        Up            20     *5/8 (12 dBm)   (52)*
+        # ab21-cap33                    5c50.1504.8de0  1       Enabled        Up            20     *7/8 (8 dBm)    (36)*
+        # ab31-cap21                    5c50.1504.8f20  1       Enabled        Up            20     *7/8 (6 dBm)    (116)*
+        # ab21-cap27                    5c50.1504.8f60  1       Enabled        Up            20     *7/8 (6 dBm)    (132)*
+        # ab21-cap35                    5c50.1504.aca0  1       Enabled        Up            20     *7/8 (6 dBm)    (140)*
+
+        # AP Name                           Mac Address     Slot    Admin State    Oper State    Width  Txpwr           Channel
+        ap_header_capture = re.compile(
+            r"^AP\s+Name\s+Mac\s+Address\s+Slot\s+Admin\s+State\s+Oper\s+State\s+Width\s+Txpwr\s+Channel$")
+        # ---------------------------------------------------------------------------------------------------------------------------------
+        delimiter_capture = re.compile(
+            r"^---------------------------------------------------------------------------------------------------------------------------------$")
+        # ab22-cap10                    5c50.1504.8be0  1       Enabled        Up            20     *6/8 (9 dBm)    (132)*
+        ap_info_capture = re.compile(
+            r"^(?P<ap_name>\S+)\s+(?P<mac_address>\S+)\s+(?P<slot>\d+)\s+(?P<admin_state>(Enabled|Disabled))\s+(?P<oper_state>\S+)\s+(?P<width>\d+)\s+(?P<tx_pwr>\*.*dBm\))\s+(?P<channel>\S+)$")
```

Comment:
can you split this line 

https://www.python.org/dev/peps/pep-0008/#:~:text=Limit%20all%20lines%20to%20a,be%20limited%20to%2072%20characters.

Reply:
Updated